ui/src/selection-panel/selection-panel-geometry-section.tsx

```diff
@@ -231,15 +233,17 @@ const SelectionPanelGeometrySection: React.FC<GeometryPlansPanelProps> = ({
                     }/${planHeaderCount})`}{' '}
                 </span>
 
-                <div>
-                    <Button
-                        size={ButtonSize.SMALL}
-                        variant={ButtonVariant.GHOST}
-                        isPressed={planDownloadPopupOpen}
-                        onClick={() => togglePlanDownloadPopupOpen(!planDownloadPopupOpen)}
-                        icon={Icons.Download}
-                    />
-                </div>
+                <PrivilegeRequired privilege={DOWNLOAD_GEOMETRY}>
```

Comment:
Täällä Slackissa puhuttu privilege-tarkastelu. Popupin sai siis aiemmin auki jos vaan oli oikeudet nähdä nappi, mutta lataaminen ei aiemminkaan toiminut ilman näitä oikkia, sillä bäkkkäripuolella oli tälle jo rajoitus